eva/models/storage/batch.py

```diff
@@ -175,6 +218,29 @@ def sort_orderby(self, by, sort_type=None) -> None:
         else:
             logger.warn("Columns and Sort Type are required for orderby")
 
+    def invert(self) -> None:
+        self._frames = ~self._frames
+
+    def all_true(self) -> bool:
+        return self._frames.all().bool()
+
+    def all_false(self) -> bool:
+        inverted = ~self._frames
+        return inverted.all().bool()
+
+    def create_mask(self) -> List:
```

Comment:
What is this?

Reply:
It is used when evaluating logical expressions.